core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -221,23 +193,23 @@ public WorkerInfo generateWorkerInfo(Set<WorkerInfoField> fieldRange, boolean is
     for (WorkerInfoField field : checkedFieldRange) {
       switch (field) {
```

Comment:
is this method called with usage and meta lock held? 
If so can you update the method comment?